tests/unit/import-account.spec.js

```diff
@@ -20,10 +23,20 @@ describe.skip('ImportAccount', () => { // TODO: rewrite test
     value: 'mystery mystery mystery mystery mystery mystery mystery mystery mystery mystery mystery mystery',
   },
   {
-    name: 'input correct phrase',
+    name: 'input correct seed phrase of 12 words ',
     value: 'media view gym mystery all fault truck target envelope kit drop fade',
     correct: true,
   },
+  {
+    name: 'input correct seed phrase of 18 words ',
+    value: 'lonely asset near deputy child echo biology talk receive pen alcohol habit myth retreat rebel grief twenty nothing',
+    correct: true,
+  },
+  {
+    name: 'input correct seed phrase of 24 words ',
+    value: 'draft cruise tenant ride extend rose urban mean sponsor quit friend indoor door mean toe donate journey minute annual rough give giggle motion zebra',
+    correct: true,
+  },
```

Comment:
For me it looks like the space at the end of the `name` is not needed
```suggestion
    name: 'input correct seed phrase of 12 words',
    value: 'media view gym mystery all fault truck target envelope kit drop fade',
    correct: true,
  },
  {
    name: 'input correct seed phrase of 18 words',
    value: 'lonely asset near deputy child echo biology talk receive pen alcohol habit myth retreat rebel grief twenty nothing',
    correct: true,
  },
  {
    name: 'input correct seed phrase of 24 words',
    value: 'draft cruise tenant ride extend rose urban mean sponsor quit friend indoor door mean toe donate journey minute annual rough give giggle motion zebra',
    correct: true,
  },
```